### PR TITLE
rm intechainadapter mentions NTRN-299

### PR DESCRIPTION
**Pull request by quasisamurai**
there are 5 PRs related to this task
1) [ICQ relayer](https://github.com/neutron-org/neutron-query-relayer/pull/72)
2) [neutron](https://github.com/neutron-org/neutron/pull/126)
3)  [integration tests](https://github.com/neutron-org/neutron-integration-tests/pull/67)
4) [dev contracts](https://github.com/neutron-org/neutron-dev-contracts/pull/5)
5) [neutron SDK](https://github.com/neutron-org/neutron-sdk/pull/76)

[TEST ACTIONS RUN](https://github.com/neutron-org/neutron-tests/actions/runs/3948296511)

### CODE REVIEW

**Review by foxpy — CHANGES_REQUESTED** [review]
Running `grep --exclude-dir=.git -R interchainadapter .` I can still see mentions of interchainadapter.

**Review by pr0n00gler — DISMISSED** [review]

File: docs/static/openapi.yml [hunk 1]
```diff
@@ -33090,7 +33090,7 @@ paths:
         - Query
   /neutron/interchainqueries/query_result:
     get:
-      operationId: NeutronInterchainadapterInterchainqueriesQueryResult
+      operationId: NeutronInterchainqueriesQueryResult
```
Comment (sotnikov-s) [inline docs/static/openapi.yml:33093]: here and below (upd: oh, and above): NeutronInterchain**q**ueries -> NeutronInterchain**Q**ueries

and please don't forget to reflect these changes in the relayer repo

File: proto/interchainqueries/genesis.proto [hunk 2]
```diff
@@ -53,8 +53,8 @@ message KVKey {
   bytes key = 2;
 }
 
-// GenesisState defines the interchainadapter module's genesis state.
-message GenesisState { 
-  Params params = 1 [ (gogoproto.nullable) = false ]; 
+// GenesisState defines the neutron module's genesis state.
```
Comment (sotnikov-s) [inline proto/interchainqueries/genesis.proto]: defines the **neutron** module's -> defines the **interchainqueries** module's

File: proto/interchaintxs/v1/genesis.proto [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-package neutron.interchainadapter.interchaintxs;
+package neutron.interchaintxs;
 
 import "gogoproto/gogo.proto";
 import "interchaintxs/v1/params.proto";
 
 option go_package = "github.com/neutron-org/neutron/x/interchaintxs/types";
 
-// GenesisState defines the interchainadapter module's genesis state.
+// GenesisState defines the neutron module's genesis state.
```
Comment (sotnikov-s) [inline proto/interchaintxs/v1/genesis.proto]: defines the **neutron** module's -> defines the **interchaintxs** module's

**Review by foxpy — CHANGES_REQUESTED** [review]
I can still see mentions of interchain adapter: `grep --exclude-dir=data --exclude-dir=.git -i -I -R adapter .`

**Review by foxpy — APPROVED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Pull request merged**